### PR TITLE
fix: adding file path of test suite to trx

### PR DESCRIPTION
**Pull request by deepakkamboj**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
-dist/
```
Comment (no23reason) [inline .gitignore:1]: We definitely want to keep the `dist` folder in gitignore. Can you please add it back?

**Comment by deepakkamboj** [comment]
Hi

I have created a new pull request will all the changes suggested by you:
https://github.com/deepakkamboj/jest-trx-results-processor/pull/5

Please commit this PR in your repository:
https://github.com/no23reason/jest-trx-results-processor

Please create a new NPM package from above pull request.

Thanks and Regards,

Deepak Kamboj
Mobile:
+1-
425-365-7211

Website: www.deepakkamboj.com
Email: deepakkamboj@gmail.com | deepak@deepakkamboj.com

**Comment by deepakkamboj** [comment]
> Thank you for your PR. Can you please address the other comen, squash your commits and use conventional commit messages as described in [CONTRIBUTING.md](https://github.com/no23reason/jest-trx-results-processor/blob/master/CONTRIBUTING.md)?

fixed

**Comment by deepakkamboj** [comment]
Closing this PR and creating new PR

**Comment by deepakkamboj** [comment]
Dear Dan,

I have created a new Pull Request for adding test suite path to codeBase
field:

https://github.com/no23reason/jest-trx-results-processor/pull/259

 Line#158
[image: image.png]
const filepath = path.relative("./", testSuiteResult.testFilePath);

 Line#168

[image: image.png]

.att("codeBase", filepath)

If you feel that PR is not in compliance with the comments.

Please create a new PR with the above changes and check-in the code.

I need these changes urgently in my project.


Thanks for all your support in advance.


Thanks and Regards,

Deepak Kamboj
Mobile:
+1-
425-365-7211

Website: www.deepakkamboj.com
Email: deepakkamboj@gmail.com | deepak@deepakkamboj.com

**Comment by deepakkamboj** [comment]
Here is latest pull request with proper comments:

https://github.com/no23reason/jest-trx-results-processor/pull/269


Thanks and Regards,

Deepak Kamboj
Mobile:
+1-
425-365-7211

Website: www.deepakkamboj.com
Email: deepakkamboj@gmail.com | deepak@deepakkamboj.com

**Pull request closed without merging**